src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,32 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList/CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <a className="UserInfo" href="mailto:Sincere@april.biz">
```

Comment:
The email address should be dynamic instead of hard-coded. Use the email from the post's user: `post.user?.email` 📧